public/js/actions/subscriptions.js

```diff
@@ -98,11 +100,20 @@ export function _createSubscription({dispatch, productId,
     console.log('Successfully subscribed + completed payment');
     dispatch(transactionActions.complete({userEmail: email}));
   }).fail($xhr => {
-    if (data.pay_method_nonce) {
-      dispatch({
-        type: actionTypes.CREDIT_CARD_SUBMISSION_ERRORS,
-        apiErrorResult: $xhr.responseJSON,
-      });
+    console.log($xhr);
+    if ($xhr.status === 400 && $xhr.responseJSON &&
+        $xhr.responseJSON.error_response) {
+      var allErrors = $xhr.responseJSON.error_response.__all__;
+      if (allErrors && hasErrorSubString(allErrors, 'already subscribed')) {
```

Comment:
Yeah I didn't want to block on that. But definitely I thought the same doing this.